eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -602,7 +601,6 @@ object Helpers {
       val channelKeyPath = keyManager.channelKeyPath(localParams, channelVersion)
       val localHtlcPubkey = Generators.derivePubKey(keyManager.htlcPoint(channelKeyPath).publicKey, remoteCommit.remotePerCommitmentPoint)
       val remoteHtlcPubkey = Generators.derivePubKey(remoteParams.htlcBasepoint, remoteCommit.remotePerCommitmentPoint)
-      val localPerCommitmentPoint = keyManager.commitmentPoint(channelKeyPath, commitments.localCommit.index.toInt)
```

Comment:
Is it expected that this value was unused? Looks like a check was removed at some point?

Reply:
Has been added by @sstone in 24dadff6253278662d95bd1bc4871cddff706fe4 when reworking the key manager and used by another variable `localRevocationPubKey` which was unused. I then cleaned up `localRevocationPubKey` in be4cf6b6940c0a8d95988420c2c24217954d106d without noticing that it was the only variable using `localPerCommitmentPoint `, and there we are...

Reply:
Ok so we can remove it safely :)